src/components/ConsoleEntry.jsx

```diff
@@ -3,17 +3,28 @@ import PropTypes from 'prop-types';
 import {ConsoleEntry as ConsoleEntryRecord} from '../records';
 import ConsoleOutput from './ConsoleOutput';
 
-export default function ConsoleEntry({entry}) {
-  const {expression} = entry;
+export default function ConsoleEntry({entry, currentCompiledProjectKey}) {
+  const {expression, evaluatedByCompiledProjectKey} = entry;
+  const isActive = currentCompiledProjectKey === evaluatedByCompiledProjectKey;
 
   return (
     <div className="console__entry">
-      <div className="console__expression">{expression}</div>
-      <ConsoleOutput entry={entry} />
+      <div
+        className={
+          isActive ? 'console__expression' : 'console__expression_inactive'
```

Comment:
The `classnames` module gives you a really nice way to do this:

```
className={
  classnames('console__expression', {console__expression_inactive: !isActive}
}
```

Note that this also always gives it the `console__expression` class—a block or element should always have the base class, with modifier classes alongside it if needed.